### ISSUE TITLE
深度学习端侧部署

### ISSUE THREAD

**Issue by kiah2008** [kiah2008/kiah2008.github.io#36]
https://www.zhihu.com/question/329372124

**Comment by kiah2008** [comment]
实战潜入式AI算法
https://zhuanlan.zhihu.com/p/70323042

**Comment by kiah2008** [comment]
qualcomm snpe-snapdragon neuron process enginine
https://developer.qualcomm.com/sites/default/files/docs/snpe/revision_history.html

**Comment by kiah2008** [comment]
Qualcomm® Hexagon™ Processor Features: Qualcomm® Hexagon™ Tensor Accelerator, Qualcomm® Hexagon™ Scalar Accelerator, Qualcomm® Hexagon™ Vector eXtensions (HVX), Fused AI Accelerator architecture

**Comment by kiah2008** [comment]
[Qualcomm AI Research](https://www.qualcomm.com/invention/artificial-intelligence/ai-research) works to advance AI and make its core capabilities – perception, reasoning, and action – ubiquitous across devices. The goal is to make breakthroughs in fundamental AI research and scale them across industries. One way we contribute innovative and impactful AI research to the rest of the community is through novel papers at academic conferences.

[Qualcomm AI Research](https://www.qualcomm.com/invention/artificial-intelligence/ai-research) works to advance AI and make its core capabilities – perception, reasoning, and action – ubiquitous across devices. The goal is to make breakthroughs in fundamental AI research and scale them across industries. One way we contribute innovative and impactful AI research to the rest of the community is through novel papers at academic conferences.

Beyond papers, the Qualcomm Innovation Center (QuIC) actively contributes code based on this breakthrough research to open source projects.

[The AI Model Efficiency Toolkit (AIMET)](https://quic.github.io/aimet-pages/index.html) is a library that provides advanced quantization and compression techniques for trained neural network models. QuIC open sourced AIMET on GitHub to collaborate with other leading AI researchers, provide a simple library plugin for AI developers, and help migrate the ecosystem toward integer inference. Read the [blog post](https://www.qualcomm.com/news/onq/2020/05/04/open-sourcing-ai-model-efficiency-toolkit) or watch some [informational AIMET videos](https://www.youtube.com/channel/UCrTwUK05waqPCU_955TeRgg) to learn more.

[The AIMET Model Zoo](https://github.com/quic/aimet-model-zoo/), another GitHub project, provides the recipe for quantizing popular 32-bit floating point (FP32) models to 8-bit integer (INT8) models with little loss in accuracy. Read the [blog post](https://www.qualcomm.com/news/onq/2021/01/22/aimet-model-zoo-highly-accurate-quantized-ai-models-are-now-available) to learn more. Check out the [Qualcomm Innovation Center YouTube](https://www.youtube.com/channel/UCrTwUK05waqPCU_955TeRgg) channel for informational videos on our open source projects to help developers get started.

Data is another crucial element for machine learning. If you need the Qualcomm Abstract Syntax Tree (QAST) dataset that was used to support the experiments in our workshop paper at ICLR 2019: Simulating Execution Time of Tensor Programs Using Graph Neural Networks, check out our [QAST Project Page](https://developer.qualcomm.com/project/qast-dataset-tensor-programs-execution-times). We hope this new dataset will benefit the graph research community and raise interest in Optimizing Compiler research.

https://github.com/quic/aimet
https://www.qualcomm.com/news/onq/2020/05/open-sourcing-ai-model-efficiency-toolkit

**Comment by kiah2008** [comment]
https://discuss.pytorch.org/c/mobile/18

**Comment by kiah2008** [comment]
https://www.tensorflow.org/lite/performance/model_optimization?hl=zh-cn

tensorflow性能优化

**Comment by kiah2008** [comment]
https://mlc.ai/

**Comment by kiah2008** [comment]
https://bj.bcebos.com/ai-studio-online/a9f5b23b56f54775907d6769a31bb83f747731c2728143078f0ee87b39f09b0c?authorization=bce-auth-v1%2F5cfe9a5e1454405eb2a975c43eace6ec%2F2022-09-04T15%3A27%3A22Z%2F-1%2F%2F84cd6c940e653040f753e1090af5ef6176cea3bb32b27cd4409dfaa7d9caef77&responseContentDisposition=attachment%3B%20filename%3DNLP%E7%9F%A5%E8%AF%86%E5%9C%B0%E5%9B%BE%E7%BA%BF%E4%B8%8A%E7%89%88V1.jpg
